### DISCUSSION TITLE
Is beets still maintained / developed ?

### DISCUSSION THREAD

**Discussion by onjour** [beetbox/beets#5136] [Q&A] [answered]
Basically the title, it's still my go-to and I wish to see it grow and prosper

**Comment by arsaboo** [comment, marked as answer]
Last PR was merged 4 days back...beets is live and well.

**Reply by onjour** [reply]
That's great news but why can't there be new releases since 2021?

**Reply by Serene-Arc** [reply]
There's been a bit of trouble doing the releases. We're moving to a system where we're automating all aspects of releasing but it's a slow process until it's actually done.

**Reply by onjour** [reply]
I appreciate your hard work but from an user perspective is there anything we can do @Serene-Arc to update our favorite tool while you sort this out?

**Reply by DjSlash** [reply]
[You can run the latest source version of beets.](https://beets.readthedocs.io/en/stable/faq.html#run-the-latest-source-version-of-beets)
